server/chat-plugins/datasearch.ts

```diff
@@ -632,12 +632,15 @@ function runDexsearch(target: string, cmd: string, canAll: boolean, message: str
 			}
 
 			if (['mono', 'monotype'].includes(toID(target))) {
+				if (singleTypeSearch === isNotSearch) return {error: "A search cannot include and exclude 'monotype'."};
+				if (parameters.length > 1) return {error: "The parameter 'monotype' cannot have alternative parameters."};
 				singleTypeSearch = !isNotSearch;
 				orGroup.skip = true;
 				continue;
 			}
 
 			if (target === 'natdex') {
+				if (parameters.length > 1) return {error: "The parameter 'natdex' cannot have alternative parameters."};
```

Comment:
Is this the case? `/ds natdex, water, earthquake, bst>600` and similar searches work fine for me.

Reply:
This is meant to prevent stuff like `/ds natdex|blue` where the natdex parameter is part of a group of alternatives

Reply:
Ah, okay.